hack/generator/pkg/jsonast/jsonast.go

```diff
@@ -580,6 +584,7 @@ func oneOfHandler(ctx context.Context, scanner *SchemaScanner, schema *gojsonsch
 			// but we still need it for controller-gen
 			jsonName := scanner.idFactory.CreateIdentifier(name, astmodel.NotExported)
 			field := astmodel.NewFieldDefinition(fieldName, jsonName, concreteType).MakeOptional()
+			field = field.WithDescription(&fieldDescription)
```

Comment:
Why not chain `.WithDescription()` after `.MakeOptional()`?

Reply:
Ah yes, now I can -- had structured it slightly differently originally so couldn't as easily. Will change